ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/UnavailableDateMapper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire.CCDUnavailableDate;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.UnavailableDate;
+
+@Component
+public class UnavailableDateMapper {
+
+    public CCDCollectionElement<CCDUnavailableDate> to(UnavailableDate unavailableDate) {
```

Comment:
same as above, I'd rather map from `CCDUnavailableDate ` to `UnavailableDate` (and viceversa)